verification/curator-service/ui/src/components/AutomatedSourceForm.tsx

```diff
@@ -135,7 +135,7 @@ export default function AutomatedSourceForm(props: Props): JSX.Element {
                             </Typography>
                         </div>
                         <Form>
-                            <fieldset className={classes.allFormSections}>
+                            <Paper className={classes.allFormSections}>
```

Comment:
you can remove the fieldset custom css that I added to index.css now I believe

Reply:
Done.